tests/macos/xcodeproj/Single-Application-Project-AllTargets.xcodeproj/project.pbxproj

```diff
@@ -465,7 +465,7 @@
 				BAZEL_BUILD_TARGET_WORKSPACE = build_bazel_rules_ios;
 				BAZEL_LLDB_INIT_FILE = "$CONFIGURATION_TEMP_DIR/Single-Application-UnitTests.lldbinit";
 				BAZEL_LLDB_SWIFT_EXTRA_CLANG_FLAGS = "-D'REQUIRED_DEFINED_FLAG=1' -D'FLAG_WITH_VALUE_ZERO=0'";
-				BAZEL_SWIFTMODULEFILES_TO_COPY = "bazel-out/ios-x86_64-min11.0-applebin_ios-ios_x86_64-dbg-ST-103f2e67c6d8/bin/tests/macos/xcodeproj/Single_Application_UnitTests.swiftmodule bazel-out/ios-x86_64-min11.0-applebin_ios-ios_x86_64-dbg-ST-103f2e67c6d8/bin/tests/macos/xcodeproj/Single_Application_UnitTests.swiftdoc bazel-out/ios-x86_64-min11.0-applebin_ios-ios_x86_64-dbg-ST-103f2e67c6d8/bin/tests/macos/xcodeproj/Single_Application_UnitTests.swiftsourceinfo";
+				BAZEL_SWIFTMODULEFILES_TO_COPY = "bazel-out/ios-sim_arm64-min11.0-applebin_ios-ios_sim_arm64-dbg-ST-103f2e67c6d8/bin/tests/macos/xcodeproj/Single_Application_UnitTests.swiftmodule bazel-out/ios-sim_arm64-min11.0-applebin_ios-ios_sim_arm64-dbg-ST-103f2e67c6d8/bin/tests/macos/xcodeproj/Single_Application_UnitTests.swiftdoc bazel-out/ios-sim_arm64-min11.0-applebin_ios-ios_sim_arm64-dbg-ST-103f2e67c6d8/bin/tests/macos/xcodeproj/Single_Application_UnitTests.swiftsourceinfo";
```

Comment:
I don't think we want the arch to change here? `x86_64` => `arm64`

Reply:
I just ran `./tests/xcodeproj-tests.sh --update`. And `arch -arch x86_64 /bin/bash -l -c "./tests/xcodeproj-tests.sh --update"` is the same 🤦🏻‍♂️. Let me see how I can fix this.